app/livechat/server/api/v1/session.js

```diff
@@ -0,0 +1,25 @@
+import { check } from 'meteor/check';
+
+import { API } from '../../../../api';
+import { Livechat } from '../../lib/Livechat';
+
+API.v1.addRoute('livechat/userLocation/:token', {
+	get() {
+		check(this.urlParams, {
+			token: String,
+		});
+
+		return Livechat.checkUserLocation(this.urlParams.token);
+	},
+});
+
+API.v1.addRoute('livechat/addLocationData', {
+	post() {
+		check(this.bodyParams, {
+			token: String,
+			location: Object,
```

Comment:
Let's describe and check the whole `location` object structure, okay?

Reply:
Cool